compoments/report-repair/summary.js

```diff
@@ -14,20 +16,34 @@ const Summary = ({values, submit, goToStep}) => {
   const repairDetailsSummary = [
     { pageName:'Where is the problem?', value: values.repairLocation?.display, link:'repair-location'},
     { pageName:'What is the problem?', value: values.repairProblem?.display, link: 'repair-problems'},
-    { pageName:'What best describes the problem?', value: values['repairProblemBestDescription']?.display, link: 'repair-problem-best-description'},
     { pageName:'Description', value: values.description?.text, link:'repair-description'}
   ]
-  if (!isMvpReleaseVersion()) {
+  if (!isMvpReleaseVersion() && repairType != 'Communal') {
```

Comment:
Now that we are no longer trying to support MVP, are we able to remove the `!isMvpReleaseVersion()` check?
```suggestion
  if (repairType != 'Communal') {
```